src/main/java/wooteco/subway/controller/LineController.java

```diff
@@ -0,0 +1,76 @@
+package wooteco.subway.controller;
+
+import java.net.URI;
+import java.util.List;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.dto.StationResponse;
+import wooteco.subway.service.LineService;
+import wooteco.subway.utils.exceptions.LineNotFoundException;
+import wooteco.subway.utils.exceptions.StationNotFoundException;
+
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping()
```

Comment:
```suggestion
    @PostMapping
```

전달할 파라미터가 없다면 괄호를 생략해도 괜찮지 않을까요 ?

Reply:
수정하였습니다!